test_runner/src/main/kotlin/ftl/gc/GcStorage.kt

```diff
@@ -22,7 +23,7 @@ import java.nio.file.Paths
 import java.util.concurrent.ConcurrentHashMap
 
 object GcStorage {
-
+    @VisibleForTesting
```

Comment:
I think that this is not true

Reply:
Right, I tested something and forgot remove it before commit. Thanks!